server-test/src/test/scala/testpkg/BuildServerTest.scala

```diff
@@ -236,6 +236,61 @@ object BuildServerTest extends AbstractServerTest {
     )
   }
 
+  // FIXME: This is WIP a version, not meant to be final!!!
+  test("{{WIP}} buildTarget/compile [Java diagnostics] clear stale warnings") { _ =>
```

Comment:
I suggest to merge this test with the one above: to test everything related to Java diagnostics in a single test.